canonical/ubuntu.com
# Refresh Pro contact us form

## Done

- Apply redesign on `__pro-contact-us-form.html`
- Ensure that the form payload submits to Marketo successfully
- Added checkbox validation and toggle checkbox visibility functions to dynamic forms 
- Fix missing form payload error

## QA

- Check out these forms:
  - [/contact-us/form?product=pro](https://ubuntu-com-14124.demos.haus/contact-us/form?product=pro)
  - [/pricing/pro#get-in-touch](https://ubuntu-com-14124.demos.haus/pricing/pro#get-in-touch)
- Check that the fields are similar to the ones on prod: https://ubuntu.com/contact-us/form?product=pro
- Fill up form and submit
- Check that payload reflects input correctly

## Issue / Card

Fixes [WD-13105](https://warthogs.atlassian.net/browse/WD-13105)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-13105]: https://warthogs.atlassian.net/browse/WD-13105?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
